Sources/MapboxSpeechCLI/main.swift

```diff
@@ -18,6 +18,11 @@ let text = CommandLine.arguments[1]
 let options = SpeechOptions(text: text)
 var speech = SpeechSynthesizer(accessToken: token)
 
+if CommandLine.arguments.count > 2 {
+    let language = CommandLine.arguments[2]
```

Comment:
Can we make this an option like `--language` instead of a positional argument? I could see there being a future need for other options to be exposed in the command line interface too.

mapbox-directions-swift’s command line tool uses the [ArgumentParser](https://github.com/apple/swift-argument-parser/) package to parse the options more cleanly. Migrating to ArgumentParser isn’t strictly necessary for this PR, but it’s an option if you prefer it to manual argument parsing.

Reply:
I totally agree, ArgumentParser is such a great command line parser. Will implement it in my next iteration.